tests/integration/components/denali-multi-select-test.js

```diff
@@ -0,0 +1,161 @@
+        {{option.text}}
+      </DenaliMultiSelect>
+    `);
+
+    await click('.selections');
+
+    assert.dom('.pane').exists('Dropdown pane is rendered');
+
+    assert.deepEqual(
+      findAll('.option').map((option) => option.labels[0].textContent.trim()),
+      ['Item 1'],
+      'DenaliMultiSelect renders a object option element'
+    );
+  });
+
+  test('it renders a selection', async function (assert) {
+    assert.expect(2);
+
+    await render(hbs`
+      <DenaliMultiSelect @options={{array "Item 1"}} @onChange={{this.onChange}} as |option|>
+        {{option}}
+      </DenaliMultiSelect>
+    `);
+
+    await click('.selections');
+
+    assert.dom('.pane').exists('Dropdown pane is rendered');
+
+    await click('.option');
+
+    assert.deepEqual(
+      findAll('.select').map((selection) => selection.textContent.trim()),
+      ['Item 1'],
+      'DenaliMultiSelect renders selected element'
+    );
+  });
+
+  test('it supports small size', async function (assert) {
+    assert.expect(1);
+
+    this.set('isSmall', 'true');
+    await render(hbs`
+      <DenaliMultiSelect @options={{array "Item 1"}} @isSmall={{this.isSmall}} as |option|>
+        {{option}}
+      </DenaliMultiSelect>
+    `);
+
+    assert
+      .dom('div.multi-select')
+      .hasClass('is-small', 'DenaliMultiSelect has a small size when `@isSmall` arg is set to true');
+  });
+
+  test('it supports inverse colors', async function (assert) {
+    assert.expect(1);
+
+    this.set('isInverse', 'true');
+    await render(hbs`
+      <DenaliMultiSelect @options={{array "Item 1"}} @isInverse={{this.isInverse}} as |option|>
+        {{option}}
+      </DenaliMultiSelect>
+    `);
+
+    assert
+      .dom('div.multi-select')
+      .hasClass('is-inverse', 'DenaliMultiSelect has inverse colors when `@isInverse` arg is set to true');
+  });
+
+  test('it searches for an element', async function (assert) {
+    assert.expect(2);
+
+    this.set('options', [{ text: 'Item 1' }, { text: 'Item 2' }, { text: 'Item 3' }]);
+    this.set('isSearchEnabled', true);
+    this.set('searchFunc', (val, target) => val.text.toLowerCase().includes(target.toLowerCase()));
+    await render(hbs`
+      <DenaliMultiSelect @options={{this.options}} @isSearchEnabled={{this.isSearchEnabled}} @searchFunc={{this.searchFunc}} as |option|>
+        {{option.text}}
+      </DenaliMultiSelect>
+    `);
+
+    await click('.selections');
+
+    assert.deepEqual(
+      findAll('.option').map((option) => option.labels[0].textContent.trim()),
+      ['Item 1', 'Item 2', 'Item 3'],
+      'DenaliMultiSelect renders selected element'
+    );
+
+    await fillIn('.search', 'item 2');
+    await triggerKeyEvent('.search', 'keyup', 13);
+
+    assert.deepEqual(
+      findAll('.option').map((option) => option.labels[0].textContent.trim()),
+      ['Item 2'],
+      'DenaliMultiSelect renders a filtered option element'
+    );
```

Comment:
Do we have test around `onChange`?

Reply:
Modified the `it renders a selection` test to test around `onChange`

Reply:
`onChange` should have its own test